src/styles/blocks/header.scss

```diff
@@ -0,0 +1,99 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+:root {
+  --header-height: 60px;
+}
+
+.roboto-medium {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
```

Comment:
Since `.roboto-medium` has the same font properties as the `html` element, you can extend this class from `html` to avoid repeating the same font properties. SCSS `@extend` directive can be used for this purpose.